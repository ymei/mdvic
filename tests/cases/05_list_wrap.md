- This list item should wrap across lines and continue with a hanging indent to match the bullet width.

1. This is an ordered list item that also wraps and aligns subsequent lines correctly.

