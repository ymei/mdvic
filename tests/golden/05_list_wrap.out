- This list item should wrap across line
  s and continue with a hanging indent t
  o match the bullet width.

1. This is an ordered list item that als
   o wraps and aligns subsequent lines c
   orrectly.

